### ISSUE TITLE
deny viewport cascading?

### ISSUE THREAD

**Issue by nelsonpecora** [nymag/responsive-filenames#23]
One of the main reasons we use `rfn` is that we can specify viewports without dealing with viewport cascading. To this end, we should have an option (`--strict`?) that will fail if two viewports overlap.

Things that would be acceptable in strict mode:

```
folder/
  all.css

folder/
  0-100.css
  100+.css

folder/
  0-600.css
  600-1024.css
  1024+.css
```

Things that would **not** be acceptable in strict mode:

```
folder/
  all.css
  0-100.css

folder/
  0-600.css
  200-600.css

folder/
  600+.css
  1024+.css
```

**Comment by cruzanmo** [comment]
When most styles are shared across viewports, it is useful to have an all.css with the shared styles and then use the viewport-specific files only where adjustments are needed.

e.g.

```
all.css
0-600.css
600-1024.css
1024+.css
```

**Comment by nelsonpecora** [comment]
Yeah but we explicitly don't want that, right? If we don't want cascading between viewports (with the tradeoffs that entails), then we should block it.

**Comment by cruzanmo** [comment]
I would consider all filenames that contain numbers to be viewport-specific, whereas all.css is not.

**Comment by nelsonpecora** [comment]
Hmm, fair point. Also the fact that certain css things cannot live inside media queries (@keyframes, etc) means we can't enforce that.
